src/lib/context.svelte.ts

```
@@ -6,50 +6,60 @@ import { getContext, onMount, setContext } from 'svelte';
 	theme: Theme;
 	controls: {
 		code: boolean;
 		mermaid: boolean;
 	};
+	inlineCitationsMode: 'list' | 'carousel';
 	animation: {
 		enabled: boolean;
 	} & StreamdownProps['animation'];
 }
-export class StreamdownContext {
+export class StreamdownContext<Source extends Record<string, any> = Record<string, any>> {
```

Biome (2.1.2)

[error] This class is unsafely merged with an interface.

The interface is declared here.

The TypeScript compiler doesn't check whether properties defined in the interface are initialized in the class.

(lint/suspicious/noUnsafeDeclarationMerging)